src/pybert/gui/view.py

```diff
@@ -503,63 +510,63 @@
                     show_border=True,
                 ),
                 VGroup(
-                    VGroup(  # CTLE
-                        Item(name="ctle_enable", label="Enable", tooltip="CTLE enable",),
-                        HGroup(  # File
-                            Item(
-                                name="use_ctle_file",
-                                label="Use",
-                                tooltip="Select CTLE impulse/step response from file.",
-                                enabled_when="ctle_file",
-                            ),
-                            Item(
-                                name="ctle_file",
-                                label="Filename",
-                                enabled_when="use_ctle_file == True",
-                                editor=FileEditor(dialog_style="open"),
-                            ),
-                            label="File",
-                            show_border=True,
-                        ),
-                        VGroup(  # Model
-                            HGroup(
+                    HGroup(
+                        VGroup(  # CTLE
+                            Item(name="ctle_enable", label="Enable", tooltip="CTLE enable",),
+                            HGroup(  # File
                                 Item(
-                                    name="peak_freq",
-                                    label="CTLE fp",
-                                    tooltip="CTLE peaking frequency (GHz)",
-                                    enabled_when="use_ctle_file == False",
+                                    name="use_ctle_file",
+                                    label="Use",
+                                    tooltip="Select CTLE impulse/step response from file.",
+                                    enabled_when="ctle_file",
```

Comment:
You can't set a file unless `use_ctle_file` is True and you set use unless there is a `ctle_file`.  Need to revert back  to letting one of them be enabled by default?  Probably the FileEditor that you can then select use or not.

Reply:
Fixed in next push.